newIDE/app/src/stories/index.js

```diff
@@ -3964,6 +3978,33 @@ storiesOf('UserPublicProfileChip', module)
     <UserPublicProfileChip user={{ id: '123', username: 'username' }} />
   ));
 
+storiesOf('ContributionsDetails', module)
+  .addDecorator(muiDecorator)
+  .add('default', () => (
+    <>
+      <ExtensionsAccordion
+        extensions={[fireBulletExtensionShortHeader, flashExtensionShortHeader]}
+        extensionError={null}
+      />
+      <ExamplesAccordion
+        examples={[geometryMonsterExampleShortHeader]}
+        exampleError={null}
+      />
+    </>
+  ))
+  .add('no contributions', () => (
+    <>
+      <ExtensionsAccordion extensions={[]} extensionError={null} />
+      <ExamplesAccordion examples={[]} exampleError={null} />
+    </>
+  ))
+  .add('with errors', () => (
```

Comment:
Love this, having even the error states will make sure we handle these properly in the interface.